### PR TITLE
Display resources in the inspector

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add a charm with resources
- click on the Resources item in the inspector
- the resources should be displayed

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6096/
Test PASSed.

File: jujugui/static/gui/src/app/components/inspector/resources/list/_list.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.inspector-resources-list {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/resources/list/_list.scss]: The new SCSS linter has 2 space indentations.

File: jujugui/static/gui/src/app/components/inspector/test-inspector.js [hunk 2]
```diff
@@ -1311,6 +1326,89 @@ describe('Inspector', function() {
...
+    const service = {
+      get: getStub
+    };
+    const appState = {
+      sectionA: {
+        metadata: {
+          activeComponent: 'resources',
+        }}};
+    const component = jsTestUtils.shallowRender(
+      <juju.components.Inspector
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/test-inspector.js]: Wow...that props list...We need to come up with a solution to this heh.

File: jujugui/static/gui/src/app/jujulib/charmstore.js [hunk 3]
```diff
@@ -504,6 +504,51 @@ var module = module;
...
+      @method _processResources
+      @param callback {Function} A callback to handle errors or accept the data
+          from the request. Must accept an error message or null as its first
+          parameter and the response data as its second.
+      @param error {String} An error message or null if no error.
+      @param data {Object} The response from the request.
+    */
+    _processResources: function(callback, error, data) {
+      if (error !== null) {
+        callback(error, null);
```
Comment (hatched) [inline jujugui/static/gui/src/app/jujulib/charmstore.js]: I'm partial to including the data we have when there is an error. Then the client of the lib can make it's own choice what to do with all the information it has.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6106/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**